_component_design/choice-tiles.md

```diff
@@ -215,20 +224,36 @@ For selected states, the iconography changes color to match the given state. In
 
 <div class="hxRow" markdown="1">
 
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## States
+
+Choice tiles can exist in an **enabled**, **disabled**, or **error** state.
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Multi-select interaction
+### Enabled
 
-**Default interaction**: When the user first views the choice tile selector, all tiles are presented in the default state unless a preferred choice is preselected.
+Choice tiles use most of the same styling that is used for components like radios and checkboxes, with the addition of the checkmark to indicate a selected tile.
 
-**After selection**: After a selection is made, the selected tile changes to the selected state and the other tiles remain in the default state.
+For selected states, the iconography changes color to match the given state. If you cannot adjust the color of an icon, be consistent with your color decision throughout the choice tile set. Do not have some icons that change color and others that do not change color. When possible, change the icon color with the selected states.
```

Comment:
maybe move the last sentence to the top? it's redundant currently, and i kinda like the last sentence phrasing better.

Reply:
Done.